src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/TournamentQueries.cs

```diff
@@ -76,30 +76,34 @@ public Tournament Execute(TournamentByGroupCriteria criteria)
                              where g.Id == criteria.GroupId
                              select t;
 
-            return tournament.Select(GetTournamentMapping()).FirstOrDefault();
+            return tournament
+                .Select(TournamentQueriesMapper.GetTournamentMapping())
+                .FirstOrDefault();
         }
 
         /// <summary>
         /// Finds Tournament by given criteria
         /// </summary>
         /// <param name="criteria"> The criteria. </param>
         /// <returns> The <see cref="Tournament"/>. </returns>
-        public List<Tournament> Execute(GetAllCriteria criteria)
+        public ICollection<Tournament> Execute(GetAllCriteria criteria)
         {
-            return _unitOfWork.Context.Tournaments.Select(GetTournamentMapping()).ToList();
+            return _unitOfWork.Context.Tournaments
+                .Select(TournamentQueriesMapper.GetTournamentMapping())
+                .ToList();
         }
 
         /// <summary>
         /// Finds List of Tournament by given criteria
         /// </summary>
         /// <param name="criteria"> The criteria. </param>
         /// <returns> The <see cref="Tournament"/>. </returns>
-        public List<Tournament> Execute(OldTournamentsCriteria criteria)
+        public ICollection<Tournament> Execute(OldTournamentsCriteria criteria)
         {
             return _unitOfWork.Context.Tournaments
-                                      .Where(t => t.IsArchived == false)
```

Comment:
Revert this change - it was intentional

Reply:
Which line?

Reply:
100. Don't you see where this coment is placed?

Reply:
ok